btmn_106.nv.json

```diff
@@ -0,0 +1,47 @@
+{
+	"_notes": ["Compiled by Horsepin."],
+	"_copyright": "Copyright (C) 2016 by Tom Collins <tom@tomlogic.com>",
+	"_license": "GNU Lesser General Public License v3.0",
+	"_endian": "big",
+	"_roms": ["btmn_106"],
+	"_fileformat": 0.1,
+	"_version": 0.1,
+	"last_game": [
+		{"start": "1479", "encoding": "bcd", "length": 5, "packed": true},
+		{"start": "1484", "encoding": "bcd", "length": 5, "packed": true},
+		{"start": "1489", "encoding": "bcd", "length": 5, "packed": true},
+		{"start": "1494", "encoding": "bcd", "length": 5, "packed": true}
+	],
+	"high_scores": [
+		{	"label": "Dark Knight", "short_label": "GC",
```

Comment:
The label for these entries should be unique and related to the short label.  The idea was that a program would use the full label when it had space, but use the short label when there wasn't as much room (like displaying scores in a launcher menu on a physical machine's DMD).